scripts/configure

```diff
@@ -58,7 +58,7 @@ if [[ -f "${STATUS_DIR}/configured" ]]; then
 else
   echo "============ CONFIGURING ============="
 fi
-echo "========= UMBREL (${NETWORK}) ==========="
+echo "========= UMBREL (${BITCOIN_NETWORK}) ==========="
```

Comment:
So this avoids a bug where empty `NETWORK` var defaults to `BITCOIN_NETWORK=mainnet` but logs empty parentheses here? Is that correct?

If that's the case maybe it would be better to just avoid duplicating `NETWORK`/`BITCOIN_NETWORK` and just do:

```bash
NETWORK="${NETWORK:-mainnet}"
```

Then use `NETWORK` everywhere and remove `BITCOIN_NETWORK`.

Reply:
The problem was just that `${NETWORK}` was never assigned and I got an error from my shell. To me it seems like `NETWORK` is only used in the configure and start scripts and all other files like `docker-compose` use `BITCOIN_NETWORK`. So if we want to consolidate `NETWORK` and `BITCOIN_NETWORK`, I'd suggest to drop `NETWORK` and use `BITCOIN_NETWORK` everywhere.

Reply:
Sounds good to me.

Reply:
I agree `BITCOIN_NETWORK ` is a better name but it's actually not simple to change.

It breaks anything external that relies on it, `umbrel-dev` for example: https://github.com/getumbrel/umbrel-dev/blob/21ac2417e687b219ef4ca629d526ca19f2660eab/Vagrantfile#L64

If we change the name we break all umbrel-dev installs. Even if we update umbrel-dev we need to tell users to upgrade and we need to specify both versions of the var incase users are developing on a branch forked from an old version of Umbrel.

Plus it breaks any existing guides/scripts/etc.

Ok, lets stick with your current solution.